src/v/kafka/server/group.cc

```diff
@@ -2254,22 +2259,49 @@ group::remove_topic_partitions(const std::vector<model::topic_partition>& tps) {
 }
 
 std::ostream& operator<<(std::ostream& o, const group& g) {
+    auto ntp = [&g] {
+        if (g._partition) {
+            return fmt::format("{}", g._partition->ntp());
+        } else {
+            return std::string("<none>");
```

Comment:
constexpr string_view? for these static strings?

Reply:
I wish but I dunno how the types would be compatible